unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneBoundariesController/SceneBoundsFeedbackStyle_Simple.cs

```diff
@@ -28,7 +28,7 @@ public void ApplyFeedback(MeshesInfo meshesInfo, bool isInsideBoundaries)
 
                 if (isInsideBoundaries && disabledRenderers.Contains(meshesInfo.renderers[i]))
                     disabledRenderers.Remove( meshesInfo.renderers[i]);
```

Comment:
you could also remove the `disabledRenderers.Contains(meshesInfo.renderers[i])` here 👀

Reply:
You are right :)